gromacs/__init__.py

```diff
@@ -242,6 +242,12 @@ def stop_logging():
     log.clear_handlers(logger)  # this _should_ do the job...
 
 
+# Try to load environment variables set by GMXRC
```

Comment:
Please rebase against develop – I had to rewrite history for the 0.5.1 release #56  in develop and this probably messed up your PR. Sorry.

It looks to me that you're trying to merge again the GMXRC loading PR #66.